build.gradle

```diff
@@ -39,7 +39,7 @@ ext {
     'maven.io_grpc_grpc_protobuf': "io.grpc:grpc-protobuf:${libraries['version.io_grpc']}",
     'maven.io_grpc_grpc_netty_shaded': "io.grpc:grpc-netty-shaded:${libraries['version.io_grpc']}",
     'maven.io_grpc_grpc_alts': "io.grpc:grpc-alts:${libraries['version.io_grpc']}",
-    'maven.io_grpc_grpc_xds': "io.grpc:grpc-xds:${libraries['version.io_grpc']}",
```

Comment:
Do we want to remove the code references to xds as well? Like [this variable name](https://github.com/googleapis/gax-java/blob/aafded4a0b779c68d0d5702b722672a0f86ccdd1/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java#L320)?